main/powertop/APKBUILD

```diff
@@ -24,11 +20,15 @@ build() {
 	make
 }
 
+check() {
+	# There is no testsuite, check the binary
+	./src/powertop --version
+}
+
```

Comment:
We have had discussions about this before and we prefer not to use a simple test like this.
Check function should always use a testsuite or else disable it. We dont want to give the impression
we test our packages but instead just do a simple version check.

Reply:
Can you update https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#options !check entry to reflect this change in policy ?

Reply:
Updated. Thanks for the review.